src/core/Modal/Modal/Modal.tsx

```diff
@@ -11,31 +11,35 @@ import { ModalContentProps } from '../ModalContent/ModalContent';
 export type ModalVariant = 'smallScreen' | 'default';
 
 export interface ModalProps {
-  /** Show or hide the modal */
+  /** Shows or hides the modal */
   visible: boolean;
-  /** Application root element id for setting aria-hidden where needed */
+  /** Application root element id for setting aria-hidden appropriately */
   appElementId: string;
-  /** Modal container div classname for custom styling. */
+  /** Custom classname to extend or customize */
   className?: string;
-  /** Modal content wrapper styles */
+  /** Modal content wrapper inline styles */
```

Comment:
This is a bit vague as well, for inline styles?

Reply:
This is now addressed in the latest commit